domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/UnusableTripNotice.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2020. MobilityData IO.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class UnusableTripNotice extends ErrorNotice {
+    public UnusableTripNotice(final String tripId) {
+        super("stop_times.txt", E_051,
+                "Trips must have more than one stop to be usable",
+                String.format("Trip with `trip_id`: `%s` should at least have two stops in stop_times.txt", tripId),
```

Comment:
@LeoFrachet suggested to add ticks around the value we refer to improve the error report legibility and ease the task of data providers when it comes to fixing issues related to their datasets. 
For example, if we do not have these ticks and the error report claims an error for trip with trip_id `trip id 00  `, it can be difficult to track down this entity given the "invisible" (space) postfixing the "visible" characters. Although this particular situation is borderline, it can happen, and that is why I put the ticks around both the field name and value.

However, I'm totally open to change this if required, for consistency we should update all notice using the same pattern.

Reply:
That's fine, it can remain as-is. I would suggest getting feedback from providers at some point to make sure the assumption is correct.